tests/integration/dbapi/async/V2/test_errors_async.py

```diff
@@ -94,31 +85,9 @@ async def test_database_not_exists(
         ) as connection:
             await connection.cursor().execute("show tables")
 
-    assert f"Database '{new_db_name}' does not exist or not authorized" in str(
-        exc_info.value
-    ), "Invalid database error message."
-
 
 async def test_sql_error(connection: Connection) -> None:
     """Connection properly reacts to SQL execution error."""
     with connection.cursor() as c:
-        with raises(OperationalError) as exc_info:
+        with raises(FireboltStructuredError):
```

Comment:
Since structured error is such a broad error class, let's also test part of the error message here. And possibly other places.

Reply:
I don't like testing error messages because they change really often. That's why I've removed them. When we'll have error implemented property (with error codes), we'll be able to define error class by the error code and test this. For now I think it's ok to just test for the error presence

Reply:
I agree, but not checking anything is not a solution here. We're checking that any error is thrown now - it could be auth error, network error, syntax, etc. We have to go with the lesser of two evils in here, which is IMO the error message check. We don't have to check the whole message, but at least a presence of a descriptive keyword. e.g. "parse" for parsing errors, etc.